Description: fix for mkdirp ≥ 1
Author: Xavier Guimard <yadd@debian.org>
Bug-Debian: https://bugs.debian.org/973109
Forwarded: no
Last-Update: 2020-10-27

--- a/scripts/rollup/utils.js
+++ b/scripts/rollup/utils.js
@@ -49,12 +49,10 @@
 
 function asyncMkDirP(filepath) {
   return new Promise((resolve, reject) =>
-    mkdirp(filepath, error => {
-      if (error) {
-        reject(error);
-        return;
-      }
+    mkdirp(filepath).then( () => {
       resolve();
+    }).catch( error => {
+      reject(error);
     })
   );
 }
